style/sass/README.md

```diff
@@ -9,7 +9,7 @@
 ## Formatting
 
 * Use the SCSS syntax.
-* Use hyphens when naming mixins, extends, classes, functions & variables: `span-columns` not `span_columns` or `spanColumns`.
+* Use hyphens when naming mixins, extends, functions & variables: `span-columns` not `span_columns` or `spanColumns`.
 * Use one space between property and value: `width: 20px` not `width:20px`.
 * Use a blank line above a selector that has styles.
 * Prefer hex color codes `#fff` or `#FFF`.
```

Comment:
What do you think about using the longhand hex value? I know it's nitpicky, but I like the consistency of always having one treatment for the declaration type.

Reply:
I don’t have a strong preference and am fine changing to longhand, but I think that’s out of scope of the PR. 

My intention with this PR was to focus on a new format and structure of how we organize the guides, but without changing any of the specific guidelines.

Reply:
Gotcha.